package/src/main/java/com/microsoft/azure/cosmos/cassandra/CosmosRetryPolicy.java

```diff
@@ -162,4 +168,42 @@ private RetryDecision retryManyTimesWithBackOffOrThrow(int retryNumber) throws I
 
         return retryDecision;
     }
-}
+
+    private RetryDecision retryManyTimesWithMsInfoOrThrow(int retryNumber, String exception) throws InterruptedException {
+
+        RetryDecision retryDecision = null;
+
+        int retryWaitTime = getRetryAfterMs(exception);
+
+        if (this.maxRetryCount == -1) {
+            Thread.sleep(retryWaitTime);
+            retryDecision = RetryDecision.retry(null);
+        } else {
+            System.out.println("retrying after "+retryWaitTime+" millseconds");
```

Comment:
Can we use System.out.println for non console apps?

Reply:
sorry, I think I copy and pasted the wrong parts.